app/assets/stylesheets/custom/code4_styling.scss

```diff
@@ -161,3 +161,7 @@ footer .logo a {
     }
   }
 }
+
+.homepage-main-content {
+  margin-top: 2rem;
```

Comment:
We usually use `$line-height` as a reference for top margins; how about `margin-top: $line-height * 1.5`?